ozark/pom.xml

```diff
@@ -7,12 +7,53 @@
         <artifactId>parent</artifactId>
         <version>1.0.0-m03-SNAPSHOT</version>
     </parent>
-    <artifactId>ozark</artifactId>
-    <packaging>jar</packaging>
-    <name>Ozark ${project.version} MVC RI</name>
+    <artifactId>javax.mvc</artifactId>
```

Comment:
But this will change the Maven artifact coordinates. This will break all existing apps that declare Ozark as a dependency. Including the tests applications in the Ozark repository. So I think we should keep the artifact name as it is today.

Anyone has a different opinion on this?